### PR TITLE
horndis: add desc, doesn't work on big sur

### PR DESCRIPTION
**Pull request by yurikoles**
Signed-off-by: Yurii Kolesnykov <root@yurikoles.com>

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.
----
Upstream issue: https://github.com/jwise/HoRNDIS/issues/132

### CODE REVIEW

File: Casks/horndis.rb [hunk 1]
```diff
@@ -5,8 +5,11 @@
   url "https://github.com/jwise/HoRNDIS/releases/download/rel#{version}/HoRNDIS-#{version}.pkg"
   appcast "https://github.com/jwise/HoRNDIS/releases.atom"
   name "HoRNDIS"
+  desc "Android USB tethering driver"
   homepage "https://github.com/jwise/HoRNDIS"
 
+  depends_on macos: ">= :big_sur"
```
Comment (miccal) [inline Casks/horndis.rb:11]: You say is does not work on Big Sur, but this makes it _only_ for Big Sur?

Comment (yurikoles) [reply Casks/horndis.rb:11]: whoops, wrong direction of comparison

**Comment by miccal** [comment]
@yurikoles any insight into the build failure here?

**Comment by yurikoles** [comment]
It seems like recent Catalina point releases are also unsupported according to https://github.com/jwise/HoRNDIS/issues/133 :(

**Review by miccal — APPROVED** [review]

**Pull request merged**